src/selectors.ts

```diff
@@ -1,8 +1,6 @@
-export function activeFeedback(state: AppState): null | FeedbackState {
-  if (!state.popup.connected) return null
-  if (!state.popup.activeTab) return null
-
-  const activeTabId = state.popup.activeTab.id
-  if (!activeTabId) return null
-  return state.feedbackByTabId[activeTabId] || null
+export function activeTab(state: AppState): TabInfo {
+  for (const tab of state.tabs.values()) {
+    if (tab.windowId === state.focusedWindowId && tab.active) return tab
+  }
```

Comment:
Sometimes tabs re-render while we're not focused on Chrome at all. This might throw an error, no?

Reply:
I think Chrome would fire the events even if it's not in focus

Reply:
If at least one tab per window need to be "active" then we're good